app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -52,7 +52,7 @@ expander:
     title: salaried teacher training
     text: |-
       <p>
-      You can apply for a salaried teacher training course. However, places on these courses are limited and very competitive.
+      You can apply for a salaried teacher training course. However, places on these courses are limited and very competitive and some salaried courses may also charge fees.
```

Comment:
```suggestion
      You can apply for a salaried teacher training course. However, places on these courses are limited and very competitive. Some salaried courses may also charge fees.
```
Worth breaking this up to avoid two `and`s?